extension/include/boost/di/extension/injections/shared_factory.hpp

```diff
@@ -30,7 +30,7 @@ template <class T>
 struct injector_rebinder<T, T> {
```

Comment:
@krzysztof-jusiak, Do you remember that this will not work for multiple interfaces?
```
di::bind<interface, implementation>().to([&](const auto& injector)
{
    return injector.template create<std::shared_ptr<implementation>>();
})
```